crates/binjs_io/src/context/format.rs

```diff
@@ -0,0 +1,6 @@
+// Fake module. This is only a placeholder for documentation.
+
+// As of this writing, doc inclusion only works on Nightly compiler.
+#![cfg_attr(feature = "unstable", doc(include = "../../../spec/context.md"))]
+
+#![cfg_attr(not(feature = "unstable"), doc = "To generate the format spec documentation, please use `cargo doc --features unstable`. This requires Rust Nightly.")]
```

Comment:
Would be great if we could emit a warning (à la C++), but I believe Rust doesn't support it.

Reply:
A warning if we're generating doc *and* not using Rust nightly?

I don't think it's really necessary. Perhaps we could drop a hyperlink to the online html, though.